streams-contrib/streams-persist-graph/src/main/java/org/apache/streams/graph/GraphHttpPersistWriter.java

```diff
@@ -31,13 +33,15 @@
 import org.apache.streams.config.ComponentConfigurator;
 import org.apache.streams.config.StreamsConfigurator;
 import org.apache.streams.core.StreamsDatum;
-import org.apache.streams.graph.neo4j.CypherGraphHelper;
```

Comment:
unfortunately, these imports create the problem.  FSF's position is that even dynamically linked binaries are considered combined works and are therefore subject to GPL. 

There is an active effort by Neo4J to come up with a viable solution.  See the following thread:

http://s.apache.org/9SS